exp-player/addon/styles/addon.scss

```diff
@@ -37,6 +37,18 @@
     margin: 5px;
 }
 
+.center {
```

Comment:
I don't think this PR needs any especially fancy review. My main suggestion is some general advice:

1. Consider whether we want these special-purpose CSS classes- especially with such generic names that might overlap with future additions.

2. Can you accomplish your styling/layout goals using existing bootstrap functionality? If not, is there a better way to keep your CSS isolated? If we eventually refactor the ISP frames into their own repo, how will we make sure that we find all the CSS styles those frames need to work?

Picking apart one big monolith will be really hard, but there's some room to plan ahead.